packages/core/src/js/colors.ts

```diff
@@ -147,7 +138,7 @@ export const colorsBackgroundLight = {
 }
 
 export const colorsBackgroundUtility = {
-  base: '#8A99A8',
+  base: '138,153,168',
```

Comment:
Breaking change for core.

I like the direction of the change. I would like to converse a bit more about timing. 

I would like to (1) focus this POC on component style overrides, (2) be comprehensive in a move toward rgb colors in core, and (3) come to an agreement on the best timing for an rgb move.

Reply:
Necessary for handling transparentize most common usage also actually reflects how this color se works.